### ISSUE TITLE
Request, dual logos

### ISSUE THREAD

**Issue by H3jake** [AndrewPaglusch/FlashPaper#162]
Hi, thanks for your awesome software, with the new update there is now dark mode. Can there be two logos one for dark mode and one for light mode?

**Comment by AndrewPaglusch** [comment]
Good idea! @barry-smithjr is this something you'd be up for doing? If not, I can do it

**Comment by Bulletoverload** [comment]
this would be a great feature. Seconded!
